src/importers/pf2eMonsterToolImport.ts

```diff
@@ -0,0 +1,267 @@
+import { Monster, Trait } from "index";
+
+
+export async function buildMonsterFromPF2EMonsterToolFile(
+    file: File
+): Promise<Monster[]> {
+    return new Promise((resolve, reject) => {
+        const reader = new FileReader();
+        reader.onload = async (event: any) => {
+            try {
+                let json = JSON.parse(event.target.result);
+                const stats: [number, number, number, number, number, number] = [
+                    getValue(json.strength),
+                    getValue(json.dexterity),
+                    getValue(json.constitution),
+                    getValue(json.intelligence),
+                    getValue(json.wisdom),
+                    getValue(json.charisma)
+                ]
+                const ac = getValue(json.ac)
+                const hp = getValue(json.hp) || 1;
+                const abilities = getAbilities(json.specials)
+                const spellcasting = getSpells(json.spelltype, json.focuspoints, json.cantriplevel, json.spelldc, json.spellattack, json.spells, json.constant)
+                const creatureType = json.creature || "Creature"
+                const monster: Monster = {
+                    layout: "Basic Pathfinder 2e Layout",
+                    name: json.name,
+                    level: creatureType + " " + json.level.toString(),
+                    size: json.size,
+                    trait_03: json.type,
+                    modifier: getValue(json.perception),
+                    perception: getPerception(json.perception),
+                    abilities_top: abilities.filter(a => a.type === ABILITY_TOP).map(a => {return {name: a.name, desc: a.desc}}),
+                    abilities_mid: abilities.filter(a => a.type === ABILITY_MID).map(a => {return {name: a.name, desc: a.desc}}),
+                    abilities_bot: abilities.filter(a => a.type === ABILITY_BOT).map(a => {return {name: a.name, desc: a.desc}}),
+                    type: json.type,
+                    subtype: "",
+                    alignment: json.alignment,
+                    ac: ac,
+                    armorclass: getACStats(ac, getValue(json.fortitude), getValue(json.reflex), getValue(json.will)),
+                    hp: hp,
+                    health: getHealthStats(hp, json.immunity, json.resistance, json.weakness),
+                    attacks: getAttacks(json.strikes),
+                    spellcasting: spellcasting,
+                    speed: json.speed || "5 feet",
+                    stats: stats,
+                    abilityMods: stats,
+                    damage_vulnerabilities: json.weakness,
```

Comment:
I don't think these are used outside of the 5e layout.

Reply:
Yeah, I mostly just added those in to conform to the `Monster` type, which requires those parameters. It didn't bother me much that they were required/vestigial, I was assuming they were mostly for searching/filtering, so filled them in just to be safe. It'd be very easy just to leave them as empty strings as well, which wouldn't break anything from what I understand.

Reply:
> which requires those parameters. It didn't bother me much that they were required/vestigial

yeah... these are vestigial. the whole `Monster` type became almost functionally useless after extending beyond 5e, I just never got around to refactoring it out.